Meta/WPT.sh

```diff
@@ -234,11 +234,16 @@ import_wpt()
 
     pushd "${LADYBIRD_SOURCE_DIR}" > /dev/null
         ./Meta/ladybird.sh build headless-browser
+        set +e
         for path in "${TESTS[@]}"; do
             echo "Importing test from ${path}"
             ./Meta/import-wpt-test.py https://wpt.live/"${path}"
+            if [ ! "$(./Meta/import-wpt-test.py https://wpt.live/"${path}")" ]; then
+                continue
+            fi
```

Comment:
Am I misreading this or does this try to import the test twice

Reply:
It definitely is

Reply:
:facepalm: